src/main.js

```diff
@@ -75,3 +77,34 @@ function isNumber(event) {
     event.preventDefault();
   }
 }
+
+function runTimer() {
+  var now = Date.now();
+  var endTime = now + currentActivity.totalSeconds * 1000;
+  var outputTime = document.querySelector('.time');
+
+  startTimerButton.disabled = true;
+  var countdown = setInterval(function() {
+    var secondsLeft = Math.floor((endTime - Date.now()) / 1000);
+    // set the innerText of the html to the below;
+    outputTime.innerHTML = convertToClock(secondsLeft);
+    console.log(convertToClock(secondsLeft));
+    if (secondsLeft <= 0) {
+      clearInterval(countdown);
+      alert('TIMES UP');
+    };
+  }, 990);
+};
+
+function convertToClock(secondsLeft) {
```

Comment:
I think if we are going to use this in our method we will be able to get rid of this. Looks good though.